### PR TITLE
219/min pods num

### PR DESCRIPTION
**Pull request by rigazilla**
This PR forbids that a cluster size is set below the minimum cluster size required to preserve data safety

### CODE REVIEW

File: deploy/cr/minimal/cr_minimal.yaml [hunk 1]
```diff
@@ -3,4 +3,5 @@ kind: Infinispan
 metadata:
   name: example-infinispan
 spec:
+  image: infinispan/server:latest
```
Comment (galderz) [inline deploy/cr/minimal/cr_minimal.yaml]: `image` is optional, so no longer a minimal example 😉. What's the reason for this change?

**Comment by rigazilla** [comment]
closing for now
